### ISSUE TITLE
Add options.reconnectTries and options.reconnectInterval server options

### ISSUE THREAD

**Issue by apapacy** [SierraSoftworks/Iridium#59]
* @param {number} [options.reconnectTries=30] Server attempt to reconnect #times
 * @param {number} [options.reconnectInterval=1000] Server will wait # milliseconds between retries

https://github.com/mongodb/node-mongodb-native/blob/2.2/lib/server.js#LC77

**Comment by SPARTAN563** [comment]
Hi @apapacy,

This functionality is already available through the `config.options` object (which mirrors the MongoDB connection options structure).

```typescript
const core = new Iridium.Core({
  host: "localhost",
  port: 27017,
  database: "iridium_test",

  options: {
    db: {
      numberOfRetries: 5,
      retryMilliseconds: 1000
    }
  }
})
```

I have updated the definitions to make sure they track 100% with the MongoDB ones and will have that released in v7.0.3 for you.

**Comment by apapacy** [comment]
Thank you
